module/Entra/AdditionalFunctions/Remove-EntraPolicy.ps1

```diff
@@ -0,0 +1,81 @@
+function Remove-EntraPolicy {
+    [CmdletBinding(DefaultParameterSetName = '')]
+    param (
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Id
+    )
+
+    PROCESS {    
+        $params = @{}
+        $customHeaders = New-EntraCustomHeaders -Command $MyInvocation.MyCommand
+        $array = ("activityBasedTimeoutPolicies",	"defaultAppManagementPolicy",	"appManagementPolicies",	"authenticationFlowsPolicy",	"authenticationMethodsPolicy",	"claimsMappingPolicies",	"featureRolloutPolicies",	"homeRealmDiscoveryPolicies",	"permissionGrantPolicies",	"tokenIssuancePolicies",	"tokenLifetimePolicies")
```

Comment:
This is not how you define an array.
Parentheses are unnecessary.

Also, give an array a meaningful name, e.g. $policyTypes.

Reply:
fixed